end_to_end/music_recommendation/02_export_feature_groups.ipynb

```diff
@@ -0,0 +1,1151 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.            "us-west-2": "174368400705",
</pre><p>Can these hard-coded region account IDs be removed / fetched programmatically? Should be able to be fetched with image_uris.retrieve() and "data-wrangler" framework</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done, used SDK function instead</p>